pages/platform/databases/databases_06_restore_backup/guide.de-de.md

```diff
@@ -0,0 +1,97 @@
+
+## Instructions
+### Step 1: Select the database service you want to restore data from
+First you need to go on the overview page of the service you want to restore the backup from.
+
+![Select service](images/service-selection.png){.thumbnail}
+
+### Step 2: Go to the backup tab
+In the tab list click on `backups`{.action}.
+
+![service overview](images/service-overview.png){.thumbnail}
+
+### Step 3: Select the backup to restore from
+Select the backup from when you want to restore from. To help you choose you have the date at which the backups have been done in the colomn "Creation date".
+Click on the `...`{.action} button corresponding to the choosen backup. Then click on `Duplicate (Fork)`{.action} to go on the configuration page of the new service.
+
+> [!warning]
+> MongoDB service has the option to restore a backup in place. Meaning restoring the backup on the same service. This option will rollback ALL data to the state it was when the backup was done, this can induce data loss.
+
+
+![backup tab](images/backup-tab.png){.thumbnail}
+
+### Step 4: Configure the target service
+As seen before, when restoring a backups you create a new separate database service on which the backup data will be imported. You are able to configure this new service as you wish.
+#### Engine
+For obvious reason you are not able to change engine, this option is greyed. Same goes for the engine version, you will be able to update engine version once the new service is running.
+
+![engine selection](images/engine-selection.png){.thumbnail}
+
+#### Plan
+When restoring a backup you can jump from service plan.
+
+![plan selection](images/plan-selection.png){.thumbnail}
+
+#### Region
+As of now we don't allow jumping region, your new service has to be on the same region as the old one.
+
+![region selection](images/region-selection.png){.thumbnail}
+
+#### Nodes
+You are able to change the node flavor of the service. Please note that you are restricted in selection depending on backup size, you can't restore a 400 GB data set on a node with only 320 GB of disk space.
+
+![node selection](images/node-selection.png){.thumbnail}
+
+#### Options
+You are able to change name.
+
+![engine selection](images/option-selection.png){.thumbnail}
+
+
+Now click on `Create a database service`{.action} and the new service will be created. Please note that depending on the backup size it can take some time before the service is available.
+
+### Step 5: Wait for service creation
+You now just have to wait for your service to be ready.
+This new service is now completly independent from the one you forked the backup from. You can safely delete the old service without impacting the new one.
+
+> [!warning]
+> The newly created service did not duplicate Ip restriction nor User from the old one. Before using your service you have to do it.
```

Comment:
Please rephrase:
The newly created service does not duplicate IP restrictions nor users which were created on the old service. You will have to recreate those before using your new service.